tests/pos/depfuntype.scala

```diff
@@ -0,0 +1,35 @@
+object Test {
+
+  trait C { type M; val m: M }
+
+  type DF = (x: C) => x.M
+  val depfun1: DF = (x: C) => x.m
```

Comment:
I like than both the type and the value have the same syntax :)

Reply:
Yes. This is something that I really like about Scala in general.